src/main/java/ca/mcgill/ecse428/potatopeeps/config/DatabaseLoader.java

```diff
@@ -95,6 +96,25 @@ public void run(String... strings) {
                     menuItemRepository.save(menuItem);
                 }
             }
+            
+            // FILL ORDER TABLE
```

Comment:
Removal of testing functionality (order's should not be pre-populated)
marking as a reminder